internal/rest/avi_obj_vs.go

```diff
@@ -991,6 +1007,10 @@ func (rest *RestOperations) AviVsVipCacheAdd(rest_op *utils.RestOp, vsKey avicac
 			vsvip_cache_obj.InvalidData = true
 		}
 
+		if resp["VsvipCloudConfigCksum"] != nil {
```

Comment:
@shubhamavi we were discussing about this. Probably it is okay to use VsvipCloudConfigCksum. But this would mean  one time PUT call. cc @sudswasavi

Reply:
one time PUT call in existing deployments right? Maybe okay for now, we need to validate this with a test.